build.sbt

```
@@ -43,7 +43,7 @@ val dependencies = Seq(
   "com.47deg"                  %% "scalacheck-toolbox-datetime" % "0.2.5" % Test,
   "org.mockito"                % "mockito-all"                  % "1.10.19" % Test,
   "org.zalando"                %% "grafter"                     % "2.6.1" % Test,
-  "io.github.embeddedkafka"    %% "embedded-kafka"              % "2.2.0" % Test
+  "io.github.embeddedkafka"    %% "embedded-kafka"              % "2.6.0" % Test
```

```suggestion
  "io.github.embeddedkafka"    %% "embedded-kafka"              % kafkaVersion % Test
```

This library usually follows the kafka version closely so I think it is fine to share the same variable

---

It does not always do that, especially with patches. so I will leave as is unless you feel strongly about it.